app/assets/javascripts/datatables.js

```diff
@@ -1,32 +1,34 @@
 $( document ).on("turbolinks:load", function() {
-  $('#passengers.admin-table').DataTable({
-    "aoColumns": [
-      null,
-      null,
-      null,
-      { "bSortable": false },
-      { "bSortable": false },
-      null,
-      { "bSortable": false },
-      { "bSortable": false },
-      { "bSortable": false },
-    ],
-    "aLengthMenu": [[25, 50, 75, 100, -1], [25, 50, 75, 100, "All"]],
-    "iDisplayLength": 100
-  });
+  if ( !$.fn.dataTable.isDataTable( '#passengers' ) ) {
```

Comment:
I know the place you copied from had space inside parentheses, but you don't need them (either set).